pkgs/top-level/perl-packages.nix

```
@@ -22206,6 +22219,18 @@ let
     };
   };
 
+  TextParsewords = buildPerlPackage {
+    pname = "Text-Parsewords";
+    version = "3.30";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/C/CH/CHORNY/Text-ParseWords-3.30.tar.gz";
+      hash = "sha256-heAjgXndQ5l+WMZr1RYRGCvH1TNQUCmi2w0yMu2v9eg=";
+    };
+    meta = {
+      license = lib.licenses.free; # Same as Perl
```

If the same as Perl license, then

```suggestion
      license = with lib.licenses; [ artistic1 gpl1Plus ];
```

LWP-UserAgent-Cached also seems to be under this scheme, so this suggestion applies there as well.

---

There should also be a `description` as well.